RELEASE_PROCESS.md

```diff
@@ -0,0 +1,38 @@
+# Release Process For AsyncAPI Specification
+
+This document is ment to explain the release process of the AsyncAPI specification. Goal of this document is to describe all details of the process so that any member of the community can jump in and help coordinating.
+
+## Release coordinator
+
+AsyncAPI specification release involves not only spec release but also the release of AsyncAPI-maintained tools, therefore it requires a single person called `release coordinator` that makes sure the release goes well through out all phases.
+
+Release coordinator doesn't have to do all the work on her/his own. Release coordinator needs to find the right people to do the work that is needed for a given phase and needs to engage as many people as needed.
+
+## Release cadence
+
+AsyncAPI releases happen on a regular basis in the following months:
+- January
+- April
+- June
+- September
+
+Regular releases concern only major and minor releases. We do not know if next release is major or minor. This is a decision that depends on the proposals for changes in the specification and how much they affect specification and tooling.
+
+Path releases are automatically released and published.
```

Comment:
What do you have in mind with `path releases`? We can have some editorials changes and in semver it can be treated as `patch`, but we don't change the spec, so we can skip the patching.

Reply:
editorial changes to spec involve changes in spec document and needs to be patched and we need to tag these changes and release and update spec on the website